inc/content_lg.php

```diff
@@ -66,6 +66,15 @@
 
 
 		<?php if ($type=='text'){ ?>
+			<div class="text-section-header-wrap">
+				<?php if ( get_sub_field ( "section_title")!=''){ ?>
```

Comment:
This part is working fine. I didn't copy/paste the code for what I sent the video on yet since it wasn't working. I'll put it in the next one for you to see.